framework/src/main/java/org/checkerframework/framework/type/treeannotator/LiteralTreeAnnotator.java

```diff
@@ -98,30 +88,30 @@ public ImplicitsTreeAnnotator(AnnotatedTypeFactory atypeFactory) {
         // For each qualifier, read the @ImplicitFor annotation and put its tree
         // classes and kinds into maps.
         for (Class<? extends Annotation> qual : quals) {
-            ImplicitFor implicit = qual.getAnnotation(ImplicitFor.class);
-            if (implicit == null) {
+            QualifierForLiterals forLiterals = qual.getAnnotation(QualifierForLiterals.class);
+            if (forLiterals == null) {
                 continue;
             }
 
             AnnotationMirror theQual =
                     AnnotationBuilder.fromClass(atypeFactory.getElementUtils(), qual);
-            for (LiteralKind literalKind : implicit.literals()) {
+            for (LiteralKind literalKind : forLiterals.value()) {
                 addLiteralKind(literalKind, theQual);
             }
 
-            for (String pattern : implicit.stringPatterns()) {
+            for (String pattern : forLiterals.stringPatterns()) {
                 addStringPattern(pattern, theQual);
             }
         }
     }
 
     /**
-     * Adds standard implicit rules. Currently sets the null literal to bottom if no other implicit
-     * is set for null literals. Also, see {@link ImplicitsTypeAnnotator#addStandardImplicits()}.
+     * Adds standard defaults. Currently sets the null literal to bottom if no other default is set
```

Comment:
I'm not sure I like the `Defaults` in the name. A literal can never specify alternative annotations, so the qualifiers given here are the effective annotations on those literals.
How about `addStandardLiteralQualifiers`?